src/main/java/net/dv8tion/jda/core/entities/impl/JDAImpl.java

```diff
@@ -637,6 +637,12 @@ public Presence getPresence()
     {
         return presence;
     }
+    
+    @Override
+    public IEventManager getEventManager()
```

Comment:
Why was this moved?

Reply:
because it wasn't in the private area, i moved it to the functions where the override functions were